docs/hyperopt.md

```diff
@@ -376,6 +381,41 @@ minimal_roi = {
     }
 ```
 
+If you are optimizing ROI, Freqtrade creates the 'roi' optimization hyperspace for you -- it's the hyperspace of components for the ROI tables. By default, each ROI table generated by the Freqtrade consists of 4 rows (steps) with the values that can vary in the following ranges:
+
+| # | minutes | ROI percentage |
+|---|---|---|
+| 1 | always 0 | 0.03...0.31 |
+| 2 | 10...40 | 0.02...0.11 |
+| 3 | 20...100 | 0.01...0.04 |
+| 4 | 30...220 | always 0 |
```

Comment:
are you sure this table is correct?

the code defines the minutes to be always between 10 and X (x beeing 40, 60 and 120)?
i suspect you wanted to change these defaults in the code ... ?

(copy-pasted from below):

```
            Integer(10, 120, name='roi_t1'),	
            Integer(10, 60, name='roi_t2'),	
            Integer(10, 40, name='roi_t3'),
```

Reply:
> are you sure this table is correct?

Yes,
please refer to `generate_roi_table()`  for how these values are used.
these are the "steps" (), the resulting value (second one, for example) is calculated as `roi_t3 + roi_t2`, so it's between 20 and 100.

> i suspect you wanted to change these defaults in the code ... ?
No, I instead wrote an adaptive roi_space() function that automatically adjusts to the ticker_interval (next PR).

Reply:
lol yes you are right - my bad, sorry.

This is 0 intuitive - we should have this documented for `roi_space()` - but i guess that'll be next PR since this will change anyway?